environments/utility/Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
 # Must match Ruby version installed in the utility image.
-ruby '>= 3.0'
+ruby '>= 3.1'
```

Comment:
You can't bump this to 3.1. The 'utility' image is based on Ubuntu 22.04 which install Ruby from APT, which is version 3.0.

In any case, changing anything in 'environments/' also requires updating image_tag.